fmriprep/workflows/bold/base.py

```diff
@@ -687,17 +703,6 @@ def init_func_preproc_wf(bold_file, has_fieldmap=False):
         ])
         # fmt:on
 
-    if nonstd_spaces.intersection(("func", "run", "bold", "boldref", "sbref")):
```

Comment:
does this mean that the boldref is now generated always? (fine by me, if so)

Reply:
No, this just ensures these inputs are always available to `func_derivatives_wf`. You still need to enable the output space to create the datasink nodes.